Modules/Core/Common/include/itkConceptChecking.h

```diff
@@ -702,13 +702,6 @@ struct HasNumericTraits
     void
     constraints()
     {
-      Detail::UniqueType<typename NumericTraits<T>::ValueType>();
```

Comment:
This is here to cause compile-time failures, if a provided custom type is missing some methods.

Reply:
Restored with comment.